backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/services/ViewCreateService.java

```diff
@@ -1009,15 +1028,24 @@ public FlowUsage createFlowUsage(Feature source, Feature target) {
         flowContainer.getOwnedRelationship().add(featureMembership);
 
         FlowUsage flowUsage = SysmlFactory.eINSTANCE.createFlowUsage();
-        this.addChildInParent(flowContainer, flowUsage);
+        this.metamodelElementMutationService.addChildInParent(flowContainer, flowUsage);
         this.elementInitializer(flowUsage);
 
-        flowUsage.getOwnedRelationship().add(this.utilService.createFlowConnectionEnd(source));
-        flowUsage.getOwnedRelationship().add(this.utilService.createFlowConnectionEnd(target));
+        // Once this method is removed, set MetamodelMutationElementService.createFlowConnectionEnd private
+        flowUsage.getOwnedRelationship().add(this.metamodelElementMutationService.createFlowConnectionEnd(source));
+        flowUsage.getOwnedRelationship().add(this.metamodelElementMutationService.createFlowConnectionEnd(target));
 
         return flowUsage;
     }
 
+    /**
+     * Deprecated. Prefer the use of :
+     *
+     * <li>
+     *     <ul>{@link org.eclipse.syson.diagram.services.DiagramMutationElementService#createBindingConnectorAsUsage(Feature, Feature, Node, Node, IEditingContext, DiagramContext)}</ul>
+     *      <ul>{@link MetamodelMutationElementService#createBindingConnectorAsUsage(Feature, Feature, Namespace)}</ul>
+     * </li>
+     */
```

Comment:
Unable to delete this method yet because it used in createPartUsageAndBindingConnectorAsUsage that do not have acces to the graphical node to find the correct owner.